capstone/docker-compose.yml

```diff
@@ -1,19 +1,49 @@
 version: '2'
 services:
-    rabbit:
-        # see https://github.com/dockerfile/rabbitmq/issues/22 if this should be persistent
-        hostname: rabbit
-        image: rabbitmq:latest
+    db:
+        image: postgres:9.6
+        restart: always
```

Comment:
Question from the peanut gallery: are you sure you want this container to restart automatically? I usually find that behavior incredibly annoying, since it means they start automatically every time you launch Docker.

Reply:
Ah, true, that's not helpful.